src/OMSimulatorLib/SystemSC.cpp

```diff
@@ -204,6 +204,7 @@ oms_status_enu_t oms::SystemSC::instantiate()
       n_states += nStates.back();
       nEventIndicators.push_back(fmus.back()->getNumberOfEventIndicators());
 
+      /* TODO: Will this memory ever get freed??? */
```

Comment:
Yes, it is done here:

https://github.com/OpenModelica/OMSimulator/blob/94de8fcd4b5dfd53bae9e6f7e4c2925bb017a347/src/OMSimulatorLib/SystemSC.cpp#L396-L403